Library/Homebrew/cmd/audit.rb

```diff
@@ -1185,6 +1185,12 @@ def audit_urls
       end
     end
 
+    # Check pypi urls
+    urls.each do |p|
+      next unless p =~ %r{^(http|https)://pypi.python.org/(.*)}i
```

Comment:
Use the simpler `%r{^https?://pypi.python.org/(.*)}`. I intentionally left out the `i` modifier because I don't see why we should be doing a case-insensitive match, but I'm happy to learn why that might be needed. (Of course the `$2` in the following line also needs to be adjusted.)